tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/IAmqpTimestamp.cs

```diff
@@ -0,0 +1,12 @@
+// <copyright file="IAmqpTimestamp.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.RabbitMQ
+{
+    internal interface IAmqpTimestamp
+    {
+        long UnixTime { get; }
+    }
```

Comment:
Using a struct is a better option if you don't need to call any methods on the type: https://github.com/DataDog/dd-trace-dotnet/blob/master/docs/development/DuckTyping.md#best-practices

Note you'd need to change references to `IAmqpTimestamp` if you accept this!

```suggestion
    [DuckCopy]
    internal struct AmqpTimestamp
    {
        public long UnixTime;
    }
```